inductiva/users/methods.py

```diff
@@ -0,0 +1,39 @@
+"""Methods to interact with the user info on the API."""
+import json
+from typing import Any, Dict, List
+
+import inductiva
+from inductiva import api
+from inductiva.client import ApiClient, ApiException
+from inductiva.client.apis.tags.users_api import UsersApi
+
+
+def _fetch_quotas_from_api() -> List[Dict]:
+    """Get information about a user's quotas.
+    """
+    api_config = api.validate_api_key(inductiva.api_key)
+
+    with ApiClient(api_config) as client:
+        api_instance = UsersApi(client)
+        try:
+            resp = api_instance.get_user_quotas().response
+
+            response_body = json.loads(resp.data.decode("utf-8"))
+
+            return response_body
```

Comment:
let's convert this to a dict of dicts

Reply:
https://github.com/inductiva/inductiva/pull/1338#discussion_r1574556449
😄